src/ConductorSharp.Toolkit/Commands/CommandRegistry.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace ConductorSharp.Toolkit.Commands
+{
+    public class CommandRegistry
+    {
+        private readonly IEnumerable<Command> commands;
+
+        public CommandRegistry(IEnumerable<Command> commands)
+        {
+            this.commands = commands;
```

Comment:
This might be a good time to align with the rest of the project and use underscore private field naming.
`private readonly IEnumerable<Command> _commands;`
and afterwards remove `this.`

Reply:
Done